web-api/src/mind.rs

```diff
@@ -235,6 +235,10 @@ impl DocumentProvider {
         self.documents.get(id)
     }
 
+    fn to_documents(&self) -> Vec<Document> {
```

Comment:
❓ it looks like most of the changes except for the `PersonalizeBy::KnnSearch` leaked into the pr? a rebase should fix that.